core/io/resource_loader.cpp

```diff
@@ -1439,6 +1441,60 @@ bool ResourceLoader::is_cleaning_tasks() {
 	return cleaning_tasks;
 }
 
+Vector<String> ResourceLoader::list_directory(const String &p_directory) {
+	RBSet<String> files_found;
+	Ref<DirAccess> dir = DirAccess::open(p_directory);
+	if (dir.is_null()) {
+		return Vector<String>();
+	}
+
+	Error err = dir->list_dir_begin();
+	if (err != OK) {
+		return Vector<String>();
+	}
+
+	String d = dir->get_next();
+	while (!d.is_empty()) {
+		bool recognized = false;
+		if (dir->current_is_dir()) {
+			if (d != "." && d != "..") {
```

Comment:
It's weird that this is needed. DirAccess' `include_navigational` is false by default. The fact that it's different between editor and export means there is some bug.

Reply:
I tried listing directory using `list_dir_begin()` in GDScript and it's correctly skipping navigational paths. Seems like `open()` in C++ works a bit differently (GDScript uses `_open()`), but not sure what causes the wrong behavior.